.eslintrc

```diff
@@ -0,0 +1,41 @@
+{
+  "parser": "babel-eslint",
```

Comment:
Not really, I just use it cause we use Babel for transpilation, didn't investigate real benefits. Do you prefer to return back to default Esprima?